### PR TITLE
update tck of kill-query

### PR DESCRIPTION
**Pull request by caton-hpg**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [x] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

### CODE REVIEW

**Comment by yixinglu** [comment]
@caton-hpg could you fix the lint error and CI ?

File: tests/tck/slowquery/KillSlowQueryBaseTest.feature [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
```
Comment (HarrisChu) [inline tests/tck/slowquery/KillSlowQueryBaseTest.feature:1]: 2022?

File: tests/tck/slowquery/PermissionViaSameService.feature [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    When executing query with user test_permission with password test:
+      """
+      USE nba;
+      SHOW QUERIES
+      | YIELD $-.SessionID AS sid, $-.ExecutionPlanID AS eid, $-.DurationInUSec AS dur
+      WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100002 STEPS"
+      | ORDER BY $-.dur
+      | KILL QUERY(session=$-.sid, plan=$-.eid)
+      """
+    Then the execution should be successful
```
Comment (HarrisChu) [inline tests/tck/slowquery/PermissionViaSameService.feature:99]: how about verify if the query existed again?
means

```
# should no result set
SHOW QUERIES| WHERE $-.DurationInUSec > 1000000 AND $-.`Query` CONTAINS "GO 100002 STEPS"
```

File: tests/tck/steps/test_kill_slow_query_via_different_service.py [hunk 3]
```diff
@@ -1,8 +1,7 @@
-# Copyright (c) 2021 vesoft inc. All rights reserved.
+# Copyright (c) 2022 vesoft inc. All rights reserved.
```
Comment (Aiee) [inline tests/tck/steps/test_kill_slow_query_via_different_service.py:1]: We only change the year in the header for newly added files.

**Review by Aiee — COMMENTED** [review]
Please add the case mentioned in the comment above.

File: tests/tck/slowquery/PermissionViaSameService.feature (hunk 2 above)
Comment (caton-hpg) [reply tests/tck/slowquery/PermissionViaSameService.feature:99]: The execution of query will take a long time.
Checking whether the query is running is on line 85.

File: tests/tck/slowquery/KillSlowQueryBaseTest.feature (hunk 1 above)
Comment (caton-hpg) [reply tests/tck/slowquery/KillSlowQueryBaseTest.feature]: done

File: tests/tck/steps/test_kill_slow_query_via_different_service.py (hunk 3 above)
Comment (caton-hpg) [reply tests/tck/steps/test_kill_slow_query_via_different_service.py]: done

**Review by HarrisChu — APPROVED** [review]

**Review by Aiee — APPROVED** [review]

**Pull request merged**